extensions/blocks/calendly/calendly.php

```diff
@@ -114,13 +115,26 @@ function load_assets( $attr, $content ) {
 				esc_attr( $classes ),
 				esc_attr( $block_id )
 			);
-			$script  = <<<JS_END
-Calendly.initInlineWidget({
-	url: '%s',
-	parentElement: document.getElementById('%s'),
-	inlineStyles: false,
-});
-JS_END;
+
+			$init_calendly_widget_js = "Calendly.initInlineWidget({
+				url: '%s',
+				parentElement: document.getElementById('%s'),
+				inlineStyles: false,
+			});";
+
+			if ( $is_p2_site ) {
```

Comment:
To do: make the same changes for the Eventbrite block.

Reply:
If this needs to be done for Eventbrite (and presumably Opentable as well as other embed blocks) should we abstract this pattern to a function? Maybe in Jetpack Gutenberg? We could add the `is_p2_site` check to one of the site helper libs too, so it could be used like `is_atomic`

Reply:
> If this needs to be done for Eventbrite (and presumably Opentable as well as other embed blocks) should we abstract this pattern to a function? Maybe in Jetpack Gutenberg? We could add the `is_p2_site` check to one of the site helper libs too, so it could be used like `is_atomic`

🤔 I moved this logic into the inline script. It could still be abstracted to a separate script, but that would mean having to import it and adding complexity.